### ISSUE TITLE
Issue with Evaluation Metrics

### ISSUE THREAD

**Issue by JLUtangchuan** [bytedance/GR-MG#6]
I'm having trouble evaluating my model using the provided checkpoint, and I noticed that the evaluation metrics are different from those reported in the original paper. Is there anything wrong with my setup or is this an inconsistency between the implementation and the paper?

|   Metric                  | 1      | 2     | 3       | 4       | 5       |
| ------------------- | ------ | ----- | ------- | ------- | ------- |
| our metric            | 0.9235 | 0.73  | 0.57425 | 0.40825 | 0.29225 |
| original metric in paper (100% data) | 0.968  | 0.893 | 0.815   | 0.727   | 0.644   |

**Comment by JLUtangchuan** [comment]
Secondly, I believe there might be a discrepancy between the goal generation model checkpoint. Has the goal gen model checkpoint fine-tuned simulation dataset? 
![image](https://github.com/user-attachments/assets/6d0556cb-90a0-422f-b99a-4eac152c64eb)

**Comment by lipeiyan-bd** [comment]
It is normal to observe some variations due to the randomness of the evaluation environment and the goal image generation model. However, the results you reproduced seem unusual. The success rate for the fifth task should not be this low. Did you ensure that you strictly followed my evaluation settings?

Additionally, I have fine-tuned the goal image generation model on CALVIN, so the generated results should be of high quality. There might be some issues with the code deployment on your machine. I suggest checking for potential bugs.

**Comment by JLUtangchuan** [comment]
Thank you for your response. I have identified the issue with the image generation—it was due to my incorrect use of the IP2P ckpt. Additionally, I have a question regarding the checkpoint you provided. I am uncertain whether it was fine-tuned using 100% of the data without including play data, or if it was fine-tuned with 10% of the data and includes play data. Your clarification on this would be greatly appreciated. Thank you!
